modules/ipi-install-configuring-the-raid.adoc

```diff
@@ -6,14 +6,21 @@
 [id="configuring-the-raid_{context}"]
 = Optional: Configuring the RAID
 
-The following procedure configures a redundant array of independent disks (RAID) during the installation process.
+The following procedure configures a redundant array of independent disks (RAID) using baseboard management controllers (BMCs) during the installation process.
 
 [NOTE]
 ====
-. {product-title} supports hardware RAID for baseboard management controllers (BMCs) using the iRMC protocol only. {product-title} {product-version} does not support software RAID.
-. If you want to configure a hardware RAID for the node, verify that the node has a RAID controller.
+If you want to configure a hardware RAID for the node, verify that the node has a supported RAID controller. {product-title} {product-version} does not support software RAID.
 ====
 
+.Hardware RAID support by vendor
+[options="header"]
+|====
+|Vendor protocol |Firmware version|RAID levels
+|Fujitsu iRMC |N/A|0, 1, 5, 6, and 10
+|Dell Redfish API with iDRAC | Version 6.10.30.20 or later | 0, 1, and 5
```

Comment:
Perhaps split into two columns for clartiy: Vendor BMC and Protocol, and let the entries be: 
Fujitsu iRMC, iRMC
Dell iDRAC, Redfish

CC: @rhjanders